### DISCUSSION TITLE
running tag searches across categories

### DISCUSSION THREAD

**Discussion by kavren** [netbox-community/netbox#10262] [Q&A]
TL;DR Is there a way for me to have it so that when I click on a Tag, that all “tagged” entries, across *all* categories, show up? By default?

The Long:

Where I work, I have some ISP connections that terminate to an ISP provided switch. I then connect whatever is the appropriate media into different ports on the ISP provided switch, and now that media is carrying one of the myriad of different connections from my ISP.

Example: ISP switch has 1 uplink connection to the World/ISP. That 1 uplink connection houses all the incoming "world data". I then connect a cat6 cable to a designated port on said ISP switch, and that cat6 carries my internet. I then connect a fiber cable to a designated port on said ISP switch, and that fiber carries 1 side of my site-to-site connection.  Each service that is being provided is assigned to a different port for me to use.

So the way I am inputting this into netbox, is that each unique service is a “circuit”. Also, I am inputting the ISP switch as a “device”. 

Now, I want to bind a “circuit” against a “interface (port)” on my ISP switch “device”. I can’t seem to find a way to do that, *BUT* I can bind a Tag against my interface, and the same tag against the circuit. This gets me mostly to where I want to go.

Here is my problem: When I go into circuits, and see the list, and see the tags… if I click on the tag I want, I get shown everything with that tag, but filtered by the category of “Circuits”. 

So that means I cannot get an entire inventory of everything with said tag, across *every* category, by just clicking on the tag from inside a category (other than the “Other” category).

When I go into the “Other” category, and click on tags section, and then click on said Tag, all entries pop! YAY!

So my questions is: Is there a way for me to have it so that when I click on a Tag, that all “tagged” entries, across *all* categories, show up? By default?

Thank you for your time.
